### DISCUSSION TITLE
im trying to decompile/analyze a file after just recently downloading it and importing it on a Ghidra fresh install, and the decompiler is not working properly. any suggestions?

### DISCUSSION THREAD

**Discussion by Tropical-Smootie** [NationalSecurityAgency/ghidra#2765] [Q&A]
Im getting these errors back:
<img width="749" alt="Screen Shot 2021-02-14 at 8 08 25 PM" src="https://user-images.githubusercontent.com/67304179/107904443-a858b100-6f00-11eb-917c-746590606f26.png">

**Comment by dev747368** [comment]
The screenshot you pasted seems to mostly be about the demangler, not the decompiler as your question title implies.  Also, you haven't included any information about your setup.

If you report this issue in the "issues" page instead of as a Discussion, it does prod you a little to include some minimal debugging information in your description, which would be very helpful.

**Reply by astrelsky** [reply]
I think it might be time to no longer append those messages to the demangler analyzer's `MessageLog` and instead only write them to `Msg`.

**Reply by dragonmacher** [reply]
I have never been fond of how those messages are displayed in that dialog.   It should be a notification that there were errors and of what type.   The log seems as good a place as any.
